### DISCUSSION TITLE
Cannot contact sales with enterprise email

### DISCUSSION THREAD

**Discussion by mortvicious** [vercel/community#3462] [Bug Report]
### Question

Greetings. I'm using .az enterprise email to contact sales, but it seems that form validation doesn't recognise email and keep throwing error: Please enter a valid email.

Any other ways to contact sales?

**Comment by ryanhemelt** [comment]
HI Mortvicious - can I have you email me at ryan.hemelt@vercel.com so we can bypass the form?
